cmd/mt-update-ttl/main.go

```diff
@@ -3,15 +3,21 @@ package main
 import (
 	"flag"
 	"fmt"
+	"log"
+	"math"
 	"os"
 	"strings"
+	"sync"
+	"sync/atomic"
 	"time"
 
 	"github.com/gocql/gocql"
 	hostpool "github.com/hailocab/go-hostpool"
 	"github.com/raintank/dur"
 )
 
+const maxToken = 9223372036854775807
```

Comment:
that's a nice way to calculate the completeness, but it took me quite a while to understand how it works, so a comment explaining what this number is and why it can be used to get the completeness based on the return of the `token()` function would be helpful

Reply:
Also could use `math.MaxInt64`